### ISSUE TITLE
Fix "bad quality" .deb package

### ISSUE THREAD

**Issue by mauriciojost** [mauriciojost/olympus-photosync#18]
A user tried to install the .deb file and it gave this error:
```
The package is of bad quality
The installation of a package which violates the quality standards isn't allowed. This could cause serious problems on your computer. Please contact the person or organisation who provided this package file and include the details beneath.
Lintian check results for /tmp/mozilla_mozillaUser0/photosync_0.5_all.deb:
E: photosync: wrong-file-owner-uid-or-gid etc/ 106/111 
E: photosync: wrong-file-owner-uid-or-gid etc/photosync/ 106/111 
E: photosync: wrong-file-owner-uid-or-gid etc/photosync/application.conf 106/111 
E: photosync: wrong-file-owner-uid-or-gid etc/photosync/log4j.properties 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/ 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/ 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/photosync/ 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/photosync/bin/ 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/photosync/bin/photosync 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/photosync/bin/photosync.bat 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/photosync/lib/ 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/photosync/lib/com.github.scopt.scopt_2.9.1-3.3.0.jar 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/photosync/lib/com.typesafe.config-1.2.1.jar 106/111 
E: photosync: wrong-file-owner-uid-or-gid opt/mauritania/photosync/lib/log4j.log4j-1.2.17.jar 106/111 
```

**Comment by mauriciojost** [comment]
Fix present in v0.6.

**Comment by mauriciojost** [comment]
Waiting for issues to be reported, or closing in 2 weeks.
